modules/node_modules/@ncigdc/components/SSM.js

```diff
@@ -210,25 +206,26 @@ const Mutation = compose(
             alignSelf: 'flex-start',
           }}
         />
-        {externalDbIds && !!Object.keys(externalDbIds).length &&
```

Comment:
you're sure that the other external reference links, Entrez Gene etc are supposed to be deleted?